### DISCUSSION TITLE
Tastensperre / keylock über Home-Assistant  aktivieren

### DISCUSSION THREAD

**Discussion by happyreacer** [OpenCCU/OpenCCU#2185] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
Ich benutze die Einbindung der RaspberryMatic in dem Home-Assistant. Ist es möglich die Tastensperre für HomeMatic Heizkörperregler HM-CC-RT-DN über ein Skript oder sowas aus dem Home-Assistant zu aktivieren?

**Comment by Baxxy13** [comment]
> die Einbindung

... nennen wir "Integration" und davon gibt es 2:

A: [die Offizielle](https://www.home-assistant.io/integrations/homematic/)
B: [die Custom "Homematic(IP) Local"](https://github.com/SukramJ/custom_homematic)

Welche meinst du?

Ist es B: dann 'sollte' es über den Service `Homematic(IP) Local: Put paramset` möglich sein.

'Sollte', weil es aktuell nicht funktioniert bzw. ich es nicht hinbekomme.
 
Wenn du B: nutzt, dann sind solche Fragen im Diskussionsteil der Integration besser aufgehoben.
Siehe: [hier](https://github.com/danielperna84/hahomematic/discussions)

Da habe ich dein Thema auch schon aufgegriffen, schauen wir mal.

**Comment by Baxxy13** [comment, marked as answer]
Tatsächlich war der Service bezüglich HM-Geräte buggy, der Maintainer hat das aber mit der Version 1.27.2 gefixt.
Das Beispiel siehst du [hier](https://github.com/danielperna84/hahomematic/issues/892)
